pillar/kingfisher_backups.sls

```diff
@@ -0,0 +1,10 @@
+postgres:
+  backup:
+    enabled: False
+    identifier: kingfisher
```

Comment:
Should `identifier` be `stanza`?

Reply:
The `identifier` is indeed the `stanza` - I felt that `identifier` made the purpose clearer to admins.
Happy with either :)

Reply:
I find it easier when a single word is used for a given concept. `stanza` is unusual, but it's what pgbackrest uses.

Reply:
Updated variable name to `stanza`